man/systemd.unit.xml

```diff
@@ -188,8 +188,8 @@
     suffix is <filename>.requires/</filename> in this case.</para>
 
     <para>Along with a unit file <filename>foo.service</filename>, a "drop-in" directory
-    <filename>foo.service.d/</filename> may exist. All files with the suffix
-    <literal>.conf</literal> from this directory will be parsed after the file itself is
+    <filename>foo.service.d/</filename> may exist. All files with the suffix <literal>.conf</literal>
+    from this directory will be parsed in order of their filename after the file itself is
```

Comment:
Thanks, I wasn’t aware that there was already some documentation on this. I’ve pushed a new version.